samples/aws/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
```

Comment:
Is this needed?

Reply:
Hey @davidfowl, I'm writing a detailed report on what I did so far. I'll answer your question

Reply:
> LocalStack Integration (@davidfowl ):
When using LocalStack, we need to run a LocalStack container before provisioning. I'm currently unsure how to do this. My next step will involve creating a ContainerResource for LocalStack. For now, I've added a docker-compose.yml file; running it first brings up LocalStack, followed by running Aws.AppHost to observe the creation of an S3 bucket.

I'll fix it in the next iteration. I'm trying to figure out how I can run a docker container in the provisioning stage.

Reply:
If local stack run as a container that has to run before your code runs, then we need to add a new life cycle event after  everything starts.